vault/logical_system_helpers.go

```diff
@@ -62,23 +67,28 @@ var (
 
 	entPaths     = entStubPaths
 	entStubPaths = func(b *SystemBackend) []*framework.Path {
-		buildEnterpriseOnlyPaths := func(paths map[string][]logical.Operation) []*framework.Path {
+		buildEnterpriseOnlyPaths := func(paths map[string]enterprisePathStub) []*framework.Path {
 			var results []*framework.Path
-			for pattern, operations := range paths {
-				operationsMap := map[logical.Operation]framework.OperationHandler{}
+			for pattern, pathSpec := range paths {
+				path := &framework.Path{
+					Pattern: pattern,
+				}
+
+				for _, parameter := range pathSpec.parameters {
+					path.Fields[parameter] = &framework.FieldSchema{
+						Type: framework.TypeString,
```

Comment:
```suggestion
						Type: framework.TypeString,
						Required: true,
```

I believe all path params must be marked as required.